### PR TITLE
Remember attribute tables in projects

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: kadas/core/CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ SET(kadas_core_HDR ${kadas_core_HDR} ${CMAKE_CURRENT_BINARY_DIR}/kadas_config.h)
 ADD_LIBRARY(kadas_core
     ${kadas_core_SRC}
     ${kadas_core_HDR}
+    kadassettingstree.h kadassettingstree.cpp
```
Comment (domi4484) [inline kadas/core/CMakeLists.txt:13]: This is probably not necessary?

**Review by domi4484 — APPROVED** [review]
Looks good to me! And cool move about the settings!

**Comment by m-kuhn** [comment]
image

**Comment by m-kuhn** [comment]
```
Couldn't load plugin 'kadas_print' due to an error when calling its classFactory() method 

NameError: name 'State' is not defined 
Traceback (most recent call last):
  File "bin\Lib\site-packages\qgis\utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "share/kadas/python/plugins\kadas_print\__init__.py", line 13, in classFactory
    from .PrintPlugin import PrintPlugin
  File "bin\Lib\site-packages\qgis\utils.py", line 892, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "share/kadas/python/plugins\kadas_print\PrintPlugin.py", line 16, in 
    from kadas.kadasgui import *
  File "bin\Lib\site-packages\qgis\utils.py", line 892, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "bin\Lib\site-packages\kadas\kadasgui\__init__.py", line 10, in 
    State.SectorStatus.HaveNothing.__doc__ = ""
    ^^^^^
NameError: name 'State' is not defined
```

**Comment by m-kuhn** [comment]
```
Couldn't load plugin 'kadas_print' due to an error when calling its classFactory() method 

NameError: name 'State' is not defined 
Traceback (most recent call last):
  File "bin\Lib\site-packages\qgis\utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "share/kadas/python/plugins\kadas_print\__init__.py", line 13, in classFactory
    from .PrintPlugin import PrintPlugin
  File "bin\Lib\site-packages\qgis\utils.py", line 892, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "share/kadas/python/plugins\kadas_print\PrintPlugin.py", line 16, in 
    from kadas.kadasgui import *
  File "bin\Lib\site-packages\qgis\utils.py", line 892, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "bin\Lib\site-packages\kadas\kadasgui\__init__.py", line 9, in 
    KadasCircularSectorItem.State.SectorStatus = State.SectorStatus
                                                 ^^^^^
NameError: name 'State' is not defined
```

**Pull request merged**